src/adapter/src/coord/command_handler.rs

```diff
@@ -755,4 +771,74 @@ impl Coordinator {
         let update = self.catalog().state().pack_session_update(session, -1);
         self.send_builtin_table_updates(vec![update]).await;
     }
+
+    fn handle_append_webhook(
+        &mut self,
+        database: String,
+        schema: String,
+        name: String,
+        tx: oneshot::Sender<Result<Option<AppendWebhookResponse>, AdapterError>>,
+    ) {
+        // Make sure the feature is enabled before doing anything else.
+        if !self.catalog().system_config().enable_webhook_sources() {
+            // We don't care if the listener went away.
+            let _ = tx.send(Err(AdapterError::Unsupported("enable_webhook_sources")));
```

Comment:
Could test that blocking this actually works.

Reply:
Added a testdrive scenario that should cover this!